django_mongodb_backend/management/commands/get_encrypted_fields_map.py

```diff
@@ -0,0 +1,45 @@
+import json
+
+from django.apps import apps
+from django.core.management.base import BaseCommand
+from django.db import DEFAULT_DB_ALIAS, connections, router
+
+
+class Command(BaseCommand):
+    help = "Generate a `schema_map` of encrypted fields for all encrypted"
+    " models in the database for use with `get_autoencryption_opts` in"
+    " production environments."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--database",
+            default=DEFAULT_DB_ALIAS,
+            help="Specify the database to use for generating the encrypted"
+            "fields map. Defaults to the 'default' database.",
+        )
+
+    def handle(self, *args, **options):
+        db = options["database"]
+        connection = connections[db]
+
+        schema_map = self.generate_encrypted_fields_schema_map(connection)
+
+        self.stdout.write(json.dumps(schema_map, indent=2))
+
+    def generate_encrypted_fields_schema_map(self, conn):
```

Comment:
There are 10 instances of `conn =` in Django, but it's far outnumbered by 86 instances of `connection =` so I'd prefer not to use the abbreviated name. I supppose I find it easier and more pleasurable not to have to read and interpret abbreviation.

Reply:
528d5038dcdc2fbabe36d143adb00377afac7d73 (we still have a couple `conn` in `base`.)